README.md

```diff
@@ -1,11 +0,0 @@
-# Codehub-Ingest (Formerly known as Hoard) ![Build Status](https://codebuild.us-east-1.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoieEg5ZDRjTUtaYkkxcTVGWFhIMEY0elFpVEhiemVsbExya2pucTJCSFZhVmFyRlFVMWNDMUF2SEFXWFhKTnYwT0NUamlBaHJxZU1WQTBOTGl6TlVXTDl3PSIsIml2UGFyYW1ldGVyU3BlYyI6ImtzODJpeUxVWnVUS0xuUEIiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=master)
-
-## Required Pre-Build Configurations
-The following environment variables need to be configured:
-1. `github_user` 
```

Comment:
Environment variables should be UPPER_SNAKE_CASE

Reply:
will do